test/spectests/spectests.cpp

```diff
@@ -210,20 +210,20 @@ class test_runner
                     const auto& expected = cmd.at("expected");
                     if (expected.empty())
                     {
-                        if (result->stack.empty())
+                        if (!result->has_value)
                             pass();
                         else
                             fail("Unexpected returned value.");
                         continue;
                     }
 
-                    if (result->stack.size() != 1)
+                    if (!result->has_value)
```

Comment:
wrong.

Reply:
Bummer, actually should have removed this part of the code with #398.